src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -172,9 +170,14 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
                     }
 
                     var location = argumentList.SyntaxTree.GetLocation(new TextSpan(argumentList.SpanStart, 0));
+
+                    var additionalProperties = new ArrayBuilder<AdditionalProperty>();
```

Comment:
Can we avoid duplicate code by having a common helper `ImmutableArray<AdditionalProperty> GetAdditionalProperties(SyntaxNode, ISyntaxFactsService)` in this type, with documentation comments explaining the current set of additional properties?